src/authentic/action_helpers.cr

```diff
@@ -35,5 +35,5 @@ module Authentic::ActionHelpers(T)
     end
   end
 
-  abstract def find_current_user(id) : T
+  abstract def find_current_user(id) : T?
```

Comment:
Yes thank you! This needed to be updated I just hand't gotten to it

Reply:
It's not mandatory, but in Crystal 0.31.0 warnings are on by default. So is a bit noisier. And eventually, this will become an error.